providers/shared/components/spa/id.ftl

```diff
@@ -46,6 +46,12 @@
                 "Description" : "The path appended to files that will be used for config",
                 "Type" : STRING_TYPE,
                 "Default" : "config"
+            },
+            {
+                "Names" : "Port",
+                "Description" : "The port used to access the SPA content.",
+                "Type" : STRING_TYPE,
+                "Default" : "https"
```

Comment:
I thin this should actually be 

```
           {
                "Names" : "Port",
                "Description" : "The port used to access the SPA content.",
                "Children" : [
                    {
                        "Names" : "HTTP",
                        "Type" : STRING_TYPE,
                        "Default" : "http"
                    },
                    {
                        "Names" : "HTTPS",
                        "Type" : STRING_TYPE,
                        "Default" : "https"
                    }
                ]
           }
```

Then the SPA should priovide 2 attribtues which match the port number from these lookups